jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/resources/META-INF/CaseMgmtorm.xml

```diff
@@ -14,8 +14,8 @@
         log.processInstanceDescription, <!-- case description -->            
         log.processId, <!--case definition  -->
         log.status, <!-- case status -->
-        log.externalId, <!-- deployment id -->      
-        log.identity, <!-- case owner -->
+        log.externalId, <!-- deployment id -->
+        (select r.entityId from CaseRoleAssignmentLog r where r.processInstanceId = log.processInstanceId and r.roleName = 'owner' and r.type = 1), <!-- case owner -->
```

Comment:
@mswiderski the idea is that owner role is always added to the case instance, independently if it was defined in the model.